content/backmatter.tex

```diff
@@ -213,99 +222,78 @@ \section{\ac{MPI} Interoperability}
 
 Another important feature of these systems is that the
 \FUNC{shmem\_pe\_accessible} routine returns \CONST{TRUE} only if the remote
 \ac{PE} is a process running from the same executable file as the local \ac{PE},
 indicating that full \openshmem support (static memory and symmetric heap) is
 available.  When using \openshmem routines within an \ac{MPI} program, the use
-of \ac{MPI} memory placement environment variables is required when using
-non-default memory placement options.
-
-\clearpage
-
-
-
+of \ac{MPI} memory-placement environment variables is required when using
+non-default memory-placement options.
 
 
 
 
 \chapter{History of OpenSHMEM}\label{sec:openshmem_history}
 
-SHMEM has a long history as a parallel programming model, having been used
-extensively on a number of products since 1993, including Cray T3D, Cray X1E,
-the Cray XT3/4, SGI Origin, SGI Altix, clusters based on the Quadrics
-interconnect, and to a very limited extent, Infiniband based clusters.
+SHMEM has a long history as a parallel-programming model and has been
+extensively used on a number of products since 1993, including the Cray T3D,
+Cray X1E, Cray XT3 and XT4, \ac{SGI} Origin, \ac{SGI} Altix, Quadrics-based
+clusters, and InfiniBand-based clusters.
 
 \begin{itemize}
-\item A SHMEM Timeline
+\item SHMEM Timeline
   \begin{itemize}
   \item Cray SHMEM
     \begin{itemize}
-    \item SHMEM first introduced by Cray Research Inc. in 1993 for Cray T3D
-    \item Cray is acquired by SGI in 1996
-    \item Cray is acquired by Tera in 2000 (MTA)
+    \item SHMEM first introduced by Cray Research, Inc.\ in 1993 for Cray T3D
+    \item Cray was acquired by \ac{SGI} in 1996
+    \item Cray was acquired by Tera in 2000 (MTA)
     \item Platforms: Cray T3D, T3E, C90, J90, SV1, SV2, X1, X2, XE, XMT, XT
     \end{itemize}
-  \item SGI SHMEM
+  \item \ac{SGI} SHMEM
     \begin{itemize}
-    \item SGI purchases  Cray Research Inc. and SHMEM was integrated into
-      SGI's Message Passing Toolkit (MPT)
-    \item SGI currently owns the rights to SHMEM and \openshmem
+    \item \ac{SGI} acquired Cray Research, Inc.\ and SHMEM was integrated into
+      \ac{SGI}'s Message Passing Toolkit (MPT)
+    \item \ac{SGI} currently owns the rights to SHMEM and \openshmem
     \item Platforms: Origin, Altix 4700, Altix XE, ICE, UV
-    \item SGI was purchased by Rackable Systems in 2009
-    \item SGI and Open Source Software Solutions, Inc. (OSSS) signed a
-      SHMEM trademark licensing agreement, in 2010
-    \end{itemize}
-  \item Other Implementations
-    \begin{itemize}
-    \item Quadrics (Vega UK, Ltd.)
-    \item Hewlett Packard
-    \item GPSHMEM
-    \item IBM
-    \item QLogic
-    \item Mellanox
-    % \item University of Houston
-    \item University of Florida
+    \item \ac{SGI} was acquired by Rackable Systems in 2009
+    \item \ac{SGI} and Open Source Software Solutions, Inc. (OSSS) signed a
+      SHMEM trademark licensing agreement in 2010
```

Comment:
Do we want to add: `\item \ac{HPE} acquires \ac{SGI} in 2016`?

Reply:
Poke @RaymondMichael

Reply:
@RaymondMichael Do you want to add this statement?

Reply:
The HPE addition looks fine, sorry I missed your question. What I question more is the oshCC -> oshc++ change. I don't know that there was anything wrong with the oshCC name, and people were used to it. Why the sudden change?

Reply:
Some platforms (e.g., OS X, IIRC) are case insensitive, so having `oshcc` and `oshCC` as separate compiler wrappers causes problems. Personally, I prefer the name `oshc++` for the C++ compiler wrapper anyway, since it seems more consistent with `gcc`/`g++` and `clang`/`clang++`, and (to me) `CC` invokes thoughts of the C compiler, not the C++ compiler.

Reply:
Thanks for the explanation.

Reply:
Yeah, git was failing in bizarre ways trying to check out the reference implementation on my mac because of the clash between oshcc and oshCC..